example_versioned_docs/version-latest/25-random.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Random
+id: random
+slug: /random
+sidebar_label: 25. Random
+description: Create random values in Wing
+keywords: [Wing language, random]
+---
+
+```js playground example title="main.w"
```

Comment:
Could be worth adding a mention that this isn't cryptographically secure (but that's ok for most applications):
````suggestion
---

The `math.random()` function can be used to create a pseudorandom number.

```js playground example title="main.w"
````

Reply:
yeah good point, added a line and warning too.